source/codegen/metadata/nidmm/enums.py

```diff
@@ -544,7 +564,7 @@
     'ThermocoupleReferenceJunctionType': {
         'values': [
             {
-                'name': 'NIDMM_VAL_FIXED',
+                'name': 'NIDMM_VAL_TEMP_REF_JUNC_FIXED',
                 'value': 2
```

Comment:
Looks like this name change is not captured in CHANGES.md

Reply:
updated changes.md with this change